db/dummydatascript/addFakeRecommendation.js

```diff
@@ -38,24 +50,37 @@ const generateExpandedWeightTable = (weightKeys) => {
 };
 
 const getRandomNumberInclusive = (begin = 0, end) => (
-  Math.floor(Math.random() * end) + begin
+  Math.floor(Math.random() * (end - begin + 1)) + begin
 );
 
 const getRandomFieldValue = weightTable => (
   weightTable[getRandomNumberInclusive(0, weightTable.length)]
 );
 
 const addRecommendationToDB = async () => {
-  const PREFERENCE_RATIO_WEIGHT_TABLE = generateExpandedWeightTable(PREFERENCE_RATIO);
+  const RECOMMENDATION_RATIO_WEIGHT_TABLE = generateExpandedWeightTable(RECOMMENDATION_RATIO);
 
   for (let i = DEFAULT_USER_NUMBER_START; i < (DEFAULT_TOTAL_USER_COUNT + DEFAULT_USER_NUMBER_START); i += 1) {
+    const randomNumber = getRandomNumberInclusive(0, RANDOM_GAME.length);
     await models.Recommendations.query().insert({
       user_id: i,
-      game_id: i,
-      title: TITLE_PREFIX + i,
-      preference: getRandomFieldValue(PREFERENCE_RATIO_WEIGHT_TABLE),
+      game_id: randomNumber,
+      title: RANDOM_GAME[randomNumber],
+      preference: getRandomFieldValue(RECOMMENDATION_RATIO_WEIGHT_TABLE),
     })
-      .then(recommendation => console.log(recommendation))
+      .then((recommendation) => {
+        elasticClient.create({
+          index: 'recommendations',
+          type: 'recs',
+          id: i,
```

Comment:
So I discovered from Shwetha that you can use the `index` function instead of `create` and let elasticsearch handle setting the `id` for you. One less thing to worry about, unless you want that level of control for setting the id that elasticsearch uses to index the item.